public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -0,0 +1,246 @@
+import { css } from '@emotion/css';
+import { debounce } from 'lodash';
+import memoizeOne from 'memoize-one';
+import React, { useEffect, useState } from 'react';
+
+import {
+  DataFrame,
+  ExploreLogsPanelState,
+  GrafanaTheme2,
+  Labels,
+  LogsSortOrder,
+  SplitOpen,
+  TimeRange,
+} from '@grafana/data/src';
+import { Themeable2 } from '@grafana/ui/src';
+
+import { fuzzySearch } from '../../../plugins/datasource/prometheus/querybuilder/components/metrics-modal/uFuzzy';
```

Comment:
As we are working on decoupling data sources, I would suggest to not import this from prometheus and maybe add it grafana/data or something.

Reply:
I copied the `fuzzySearch` for now into utils